README.md

```diff
@@ -105,12 +105,12 @@ matrices with observations stored in rows are also supported via the argument `d
 
 #### Computing column-wise and pairwise distances inplace
 
-If the vector/matrix to store the results are pre-allocated, you may use the storage (without creating a new array) using the following syntax (`i` being either `1` or `2`):
+If the vector/matrix to store the results are pre-allocated, you may use the storage (without creating a new array) using the following syntax (`dims` being either `1` or `2`):
```

Comment:
I guess we don't need to specify `dims` here now, it was just required due to the deprecation.